framework/assets/yii.validation.js

```diff
@@ -236,6 +236,23 @@ yii.validation = (function ($) {
             }
         },
 
+        _trim: function(value, options = {skipOnEmpty: true}) {
+            if (options.skipOnEmpty && pub.isEmpty(value)) {
```

Comment:
Likely it breaks behavior in `if (options.strict && value !== undefined || !options.strict && !pub.isEmpty(isString ? pub._trim(value) : value)) {`. Previously it wasn't checking for `options.skipOnEmpty` at all.

Reply:
@samdark "early return" as this called Alex, it's not change behavior